packages/wallet/src/__stories__/dummy-wallet-states.ts

```diff
@@ -6,8 +6,8 @@ const {
   channelNonce,
   libraryAddress,
   participants,
-  preFundCommitment1,
-  preFundCommitment2,
+  preFundCommitment0: preFundCommitment1,
+  preFundCommitment1: preFundCommitment2,
```

Comment:
I'm guessing that you didn't mean to do this here?

Reply:
This is the result of symbol renaming mock `preFundCommitment1` and `preFundCommitment2` to `preFundCommitment0` and `preFundCommitment1` respectively. So this was intentional. Though the right approach in this particular file is to rename the local variables as well.